package.json

```diff
@@ -0,0 +1,33 @@
+{
+    "name": "@chakrasandwich/react-shocked-pikachu",
+    "version": "1.0.2",
+    "description": "Inserts a pikachu ascii art into your react component",
+    "license": "MIT",
+    "repository": "nmestrada",
+    "main": "build/index",
+    "scripts": {
+        "start": "webpack --watch",
+        "build": "webpack"
+    },
+    "keywords": [
+        "pikachu",
+        "npm",
+        "package",
+        "chakrasandwich"
+    ],
+    "dependencies": {},
```

Comment:
At least `react` would need to be in the `dependencies` - your package relies on it in doing its work in production (i.e. "the real world", imported in someone else's cod). The `dev-dependencies` are only meant for when you're developing/testing your package.